packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -384,12 +368,18 @@ export default function MainFeedLayout({
           onRequestClose={() => setIsFeedFiltersOpen(false)}
         />
       )}
-      {createMyFeed && (
+      {isCreateMyFeedOpen && (
```

Comment:
I find the names `CreateMyFeed` and `Onboarding` quite confusing. May I suggest `LegacyOnboarding` and `Onboarding`?